index.html

```diff
@@ -441,6 +441,171 @@ <h2><dfn>GamepadEvent</dfn> Interface</h2>
       </dl>
     </section>
 
+    <section>
+      <h2><dfn>GamepadButtonEvent</dfn> Interface</h2>
+      <dl title='[Constructor(GamepadButtonEventInit eventInitDict)] interface GamepadButtonEvent : Event'
+          class='idl'>
+        <dt>readonly attribute Gamepad gamepad</dt>
+
+        <dd>
+
+            The single gamepad attribute provides access to the associated
+            gamepad data for this event.
+
+        </dd>
+
+        <dt>readonly attribute GamepadButton button</dt>
```

Comment:
Firefox's implementation uses a button index instead of a GamepadButton:

gamepadbuttondown { target: Window → /, isTrusted: true, **button: 1**, gamepad: Gamepad, currentTarget: Window → /, eventPhase: 2, bubbles: false, cancelable: false, defaultPrevented: false, composed: false, timeStamp: 1530125797442210 }

I think it would be better to include the button index to make it easier for listeners to know which button was pressed. The GamepadButton can be accessed through the gamepad attribute once we know the index, but it's difficult to determine a button's index from a GamepadButton.